charts/seed-monitoring/charts/grafana/dashboards/operators/apiserver-watches-dashboard.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
Does it make sense to rename this file to `kubernetes-api-server-watches-dashboard.json` to keep it consistent with the other dashboard files?